src/platform/macos.rs

```diff
@@ -195,6 +313,17 @@ pub trait EventLoopExtMacOS {
     /// [`run`](crate::event_loop::EventLoop::run) or
     /// [`run_return`](crate::platform::run_return::EventLoopExtRunReturn::run_return)
     fn enable_default_menu_creation(&mut self, enable: bool);
+
+    /// Adds a new callback method for the application delegate.
+    ///
+    /// ### Safety
+    /// As the underlying `add_method` documentation writes:
+    /// > Unsafe because the caller must ensure that the types match those that are expected when the method is invoked from Objective-C.
+    unsafe fn add_application_method<F: DelegateMethod>(
+        &mut self,
+        sel: Sel,
```

Comment:
Let's not expose `objc` as a public dependency; use `sel: *const c_void` instead.

It may be updated, and then users would have to always use the same version that `winit` uses. Also, it would now be a breaking change to update our version of `objc` (or swap it out with something else like [my fork](https://github.com/madsmtm/objc2), though I realise that's an argument that mostly benefits me :grin:).